docs/getting-started/installing-solidus.mdx

```diff
@@ -72,26 +75,34 @@ Solidus [in an existing app](#in-an-existing-app).
 If you have an existing Ruby on Rails application, installing Solidus is fairly simple. In your CLI:
 
 ```bash
-$ bundle add 'solidus'
-# Add `--help` to learn about all available options
+$ bundle add solidus
 $ bin/rails generate solidus:install
 ```
 
+:::info
+
+This command will install Solidus in your Rails application with the default option, or as we will see
+later, will prompt you to choose some options. If you want to see all the installation options
+offered by Solidus, run `bin/rails generate solidus:install --help`.
```

Comment:
```suggestion
If you want to see all the installation options offered by Solidus, run `bin/rails generate solidus:install --help`.
```

I find the first part a bit confusing, as it seems to convey that two different thing can happen. As we already have "The installer will prompt you on a few questions before completing the
installation:" just after, maybe there's no need to repeat it here.

Reply:
Right! I actually had this part somewhere else at the beginning and after I moved it, I didn't notice it ended up so close to that next paragraph.